samples/Backend on Ruby On Rails.md

```diff
@@ -76,15 +78,16 @@ Add a chunks controller
 	          #Deleting chunk
 	          FileUtils.rm "#{dir}/#{params[:resumableFilename]}.part#{i}", :force => true 
 	        end
-	        puts "File saved to #{dir}/#{params[:resumableFilename]}"
```

Comment:
Actually it's can be used only outside of `File.open("#{dir}/#{params[:resumableFilename]}","a")` block. In other way sometimes file becomes broken